store/balance.go

```diff
@@ -22,40 +22,51 @@ import (
 	"math/big"
 
 	gethCommon "github.com/ethereum/go-ethereum/common"
-	"github.com/getamis/eth-indexer/common"
 	"github.com/getamis/eth-indexer/model"
 	"github.com/getamis/sirius/log"
+	"github.com/shopspring/decimal"
 )
 
 var ErrInvalidBalance = errors.New("invalid balance")
 
 func (srv *serviceManager) GetBalance(ctx context.Context, address gethCommon.Address, blockNr int64) (balance *big.Int, blockNumber *big.Int, err error) {
 	logger := log.New("addr", address.Hex(), "number", blockNr)
-	// Find header
-	var hdr *model.Header
-	if common.IsLatestBlock(blockNr) {
-		hdr, err = srv.FindLatestBlock()
-	} else {
-		hdr, err = srv.FindBlockByNumber(blockNr)
+	account, err := srv.FindAccount(model.ETHAddress, address, blockNr)
+	if err != nil {
+		logger.Error("Failed to find account", "err", err)
+		return nil, nil, err
+	}
+	var ok bool
+	balance, ok = new(big.Int).SetString(account.Balance, 10)
```

Comment:
@tailingchen and I are thinking that indexer can keep returning raw data for good, and let the caller (e.g., explorer) decide the format itself and its processing. It should be good for indexer to return clean data without format interrupting.

I have another PR on explorer side which is doing the format transforming, please help to check. Thanks! :)

Reply:
@jeddie-yhc 
what is the advantage of returning raw data here?
indexer is a stand-alone open sourced project. we should try to preserve a complete/functional interface. either we move service_store.go to explorer, or we think about indexer as a service on its own.